be/src/exec/broker_reader.cpp

```diff
@@ -43,7 +43,7 @@ BrokerReader::BrokerReader(
             _path(path),
             _cur_offset(start_offset),
             _is_fd_valid(false),
-            _eof(false),
+            _file_size(0),
```

Comment:
why remove _eof? 
I think you'd better not change old code if you have sufficient reason, because this can make review work hard and easy to lead error

Reply:
I think that the member **_eof**  is not useful. Because it has exist the input pararm eof of the function **read**

Reply:
no, this _eof is indicate that all data has been read